src/Service/Storage/Ef/DbGlobalContext.cs

```diff
@@ -36,7 +38,8 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             b.HasKey(x => new { x.Tenant, x.DescriptorId });
             b.Property(x => x.DescriptorTransports).HasConversion(
```

Comment:
We don't have to change anything here. But next time we are storing multiple compile time constants (such as an enum) to the database, we should consider using the `[Flags]` attribute which will store the value as an integer.  Each enum would then have a binary representation for values.